modules/apps/27-interchain-accounts/types/errors.go

```diff
@@ -13,8 +13,8 @@ var (
 	ErrInvalidRoute                = sdkerrors.Register(ModuleName, 7, "invalid route")
 	ErrInterchainAccountNotFound   = sdkerrors.Register(ModuleName, 8, "interchain account not found")
 	ErrInterchainAccountAlreadySet = sdkerrors.Register(ModuleName, 9, "interchain account is already set")
-	ErrActiveChannelNotFound       = sdkerrors.Register(ModuleName, 10, "no active channel for this owner")
-	ErrActiveChannelAlreadySet     = sdkerrors.Register(ModuleName, 11, "active channel already set for this owner")
+	ErrActiveChannelAlreadySet     = sdkerrors.Register(ModuleName, 10, "active channel already set for this owner")
+	ErrActiveChannelNotFound       = sdkerrors.Register(ModuleName, 11, "no active channel for this owner")
```

Comment:
as a side note, this is state machine breaking (ABCI code changes), so I'd generally refrain from reorganizing error code ordering. It is hard to come up with any logical ordering when the code are grouped linearly

Reply:
That is an excellent point